src/playdate/bindings/sound.nim

```diff
@@ -25,7 +25,7 @@ type PlaydateSoundFileplayer {.importc: "const struct playdate_sound_fileplayer"
     getLength {.importc: "getLength".}: proc (player: FilePlayerPtr): cfloat {.cdecl, raises: [].}
     setOffset {.importc: "setOffset".}: proc (player: FilePlayerPtr; offset: cfloat) {.
         cdecl, raises: [].}
-    # setRate* {.importc: "setRate".}: proc (player: ptr FilePlayer; rate: cfloat) {.cdecl.}
+    # setRate {.importc: "setRate".}: proc (player: FilePlayerPtr; rate: cfloat) {.cdecl.}
```

Comment:
This change obviously doesn't do anything, but it probably points (pun intended) in the right direction